util/build-gnu.sh

```diff
@@ -265,6 +265,9 @@ sed -i -E "s|^([^#]*2_31.*)$|#\1|g" tests/printf/printf-cov.pl
 
 sed -i -e "s/du: invalid -t argument/du: invalid --threshold argument/" -e "s/du: option requires an argument/error: a value is required for '--threshold <SIZE>' but none was supplied/" -e "/Try 'du --help' for more information./d" tests/du/threshold.sh
 
+# clap returns 2 when rejecting a value
+sed -i -e "s|returns_ 1 ls |returns_ 2 ls |g" tests/ls/classify.sh
```

Comment:
Hm, I'm not sure about this change. I think we solve this differently in other tools and return `1` there. For example, `cargo run nl --number-format=invalid` returns a status code of `1`.

Reply:
sure
Do you know how to do it ?  :)

Reply:
See https://github.com/uutils/coreutils/pull/5446